components/org.wso2.carbon.identity.oauth/src/test/java/org/wso2/carbon/identity/oauth2/token/AccessTokenIssuerTest.java

```diff
@@ -111,681 +96,675 @@ public class AccessTokenIssuerTest extends PowerMockIdentityBaseTest {
     private static final String[] SCOPES_WITHOUT_OPENID = new String[]{"scope1", "scope2"};
     private static final String[] SCOPES_WITH_OPENID = new String[]{"scope1", OAuthConstants.Scope.OPENID};
 
-    @BeforeMethod
```

Comment:
These tests has been disabled for quite some time. Hence didn't put much effort to refactor these tests in this effort. Let's validate these tests in a different effort and see if we can enable them again or we have to get rid of them.